src/invite/Invite.tsx

```diff
@@ -0,0 +1,108 @@
+import { isNil, noop } from 'lodash'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, ScrollView, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import Button, { BtnSizes } from 'src/components/Button'
+import Touchable from 'src/components/Touchable'
+import ShareIcon from 'src/icons/Share'
+import Times from 'src/icons/Times'
+import { inviteModal } from 'src/images/Images'
+import { noHeader } from 'src/navigator/Headers'
+import { navigateBack } from 'src/navigator/NavigationService'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+
+import { useShareUrl } from './hooks'
+
+function Header() {
+  return (
+    <View style={styles.headerContainer}>
+      <Touchable onPress={navigateBack} borderless={true} hitSlop={variables.iconHitslop}>
+        <Times />
+      </Touchable>
+    </View>
+  )
+}
+
+function Content() {
+  const { t } = useTranslation()
+  const shareUrl = useShareUrl()
+
+  return (
+    <View style={styles.outerContentContainer}>
+      <View style={styles.innerContentContainer}>
+        <Image style={styles.art} source={inviteModal} />
+        <Text style={styles.title}>{t('inviteWithUrl.title')}</Text>
+        <Text style={styles.body}>{t('inviteWithUrl.body')}</Text>
+        <Button
+          testID="invite"
+          iconPositionLeft={false}
+          icon={<ShareIcon height={24} color="white" />}
+          text={t('inviteWithUrl.button')}
+          size={BtnSizes.SMALL}
+          disabled={isNil(shareUrl)}
+          onPress={shareUrl ?? noop}
+        />
+      </View>
+    </View>
+  )
+}
+
+export default function Invite() {
```

Comment:
![Simulator Screen Shot - iPhone 11 - 2022-09-26 at 09 41 00](https://user-images.githubusercontent.com/20150449/192220451-2b323102-a707-47e8-8788-52f3b7d5eb43.png)

hmm, the styles don't look quite right?

also, how do you feel about removing the abstractions for Header and Content? in my opinion these abstractions don't really make the component easier to understand and i think you can simplify the component / styles if the abstractions were removed. but as i am typing this i remember that actually the InviteOptionsModal has the [same design](https://www.figma.com/file/8ig95B4sfsYIDIpg3JGP03/Phone-Verification?node-id=1378%3A21235)  - i think it makes sense to reuse the same UI components. i will abstract out the UI code, i should have done this before :)

Reply:
What device is this on? I'm not seeing styles like that, but I've been testing on a large iOS simulator and Android device.

I don't have a super strong opinion on Header and Content. It's just a minor code organization pattern that I find helpful. I can squash it into one component if you want.

Reply:
OK, I've swapped out my UI for the `InviteModal` component. I think the tests I had already written are now _less_ relevant, but I left them since they do still add a little value. Let me know if you have any concerns with the changes I made.